roles/pulp/tasks/install.yml

```diff
@@ -99,7 +99,7 @@
         state: directory
         owner: '{{ pulp_user }}'
         group: '{{ pulp_group }}'
-        mode: 0775
+        mode: "u+rwx,g+rwx,o+rx"
```

Comment:
I see the original change was 0775 so this looks equivalent, but I'm concerned about the 5 on the end. I expected Pulp would work with 770. What do you all think?

Reply:
I thought this was for `/var/lib/pulp/` not `/var/lib/pulp/tmp` so I'm less concerned. I agree this is what we have documented so +1 to this PR.

Reply:
We should try to remove that permission if we don't require it, but that is a separate piece of work entirely.